src/github.com/couchbase/sync_gateway/rest/changes_api.go

```diff
@@ -478,3 +478,14 @@ func sequenceFromString(str string) uint64 {
 	}
 	return seq
 }
+
+
+func unmarshalJSONString(s string) string {
+	var js string
+	err := json.Unmarshal([]byte(s), &js); if err != nil {
+		return s
+	} else {
+		return js
```

Comment:
I was trying to work out the benefit of this approach, versus just using unmarshalling to a SequenceId, since SequenceId.UnmarshalJSON does some similar things (unmarshals to string, does ParseSequenceId).  It looks like the difference is the existing SequenceId.UnmarshalJSON doesn't handle a non-JSON string that's not a simple int (like since=10:25).   Was that your take as well?

Based on that, I feel like it might be nice to have a set of new explicit unit tests to cover the various since  query parameter permutations (123, "123", 1:2, "1:2") to make it clear that they are all handled.